hack/stabilization-changes.py

```diff
@@ -89,6 +89,8 @@ def sem_ver_less_than(a, b):
         return True
     if b_groups['prerelease'] and not a_groups['prerelease']:
         return False
+    if not (a_groups['prerelease'] or b_groups['prerelease']):
+        return False
```

Comment:
What is this case about?  Releases that only differ by `buildmetadata`?

Reply:
This handles equality when prerelease is not present. We need to handle equality because adding candidates to the list of releases considered includes the candidate currently processed. Without equality the script failed with `KeyError` on the `return` line that indexes with `prerelease`.

I'll add a comment.